src/mame/includes/hng64.h

```diff
@@ -61,6 +63,7 @@ struct polygon
 
 	float faceNormal[4];        // Normal of the face overall - for calculating visibility and flat-shading...
 	int visible;                // Polygon visibility in scene
+	int flatShade;              // Flat shaded polygon, no texture, no lighting
```

Comment:
Is there a reason not to use `bool` for this?

Reply:
No, but visible also seemed like it should be a bool

Reply:
the code is very old, IIRC at the time it was written bools were discouraged.

Reply:
Yes, this is true. Given the heavy emphasis on matching the existing code style, it's fine for it to be an int for now. However, there should be a GitHub issue filed as a to-do reminder to have a pass over HNG64 to bring it up to somewhat modern standards. I might even take a shot at it tonight. But it shouldn't be a blocking issue for merging this PR.